### PR TITLE
Free stroage test 

### PR DESCRIPTION
**Pull request by shalinikum**
https://github.com/0chain/system_test/issues/657

### CODE REVIEW

File: tests/api_tests/config/api_tests_config.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-block_worker:  https://dev.zus.network/dns
-0box_url: https://0box.dev.devnet-0chain.net
+block_worker:  https://demo.zus.network/dns
+0box_url: https://0box.demo.zus.network
```
Comment (stewartie4) [inline tests/api_tests/config/api_tests_config.yaml:2]: revert

File: tests/api_tests/0box_free_storage_test.go [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+			"blimp")
+		require.NoError(t, err)
+		require.Equal(t, 200, response.StatusCode(), "Response status code does not match expected. Output: [%v]", response.String())
+
+		marker, markerResponse, err := UnmarshalMarkerData(storageMarker)
+		require.Nil(t, err)
+		require.Equal(t, X_APP_CLIENT_ID, marker.Recipient)
+		require.Equal(t, marker.Assigner, "0chain")
+		require.Equal(t, markerResponse.RecipientPublicKey, X_APP_CLIENT_KEY)
+		require.Positive(t, marker.FreeTokens)
```
Comment (Kishan-Dhakan) [inline tests/api_tests/0box_free_storage_test.go:67]: There should also be a task_id being returned here, which needs to be used to poll the status of the funding task. In the polling, once response contains `funded` as `true`, you need to redeem the marker for allocation to be created. Also, confirm allocation is created from sharders.

**Review by Kishan-Dhakan — CHANGES_REQUESTED** [review]
Happy with the changes so far, needs some extension to fully cover the flow

File: tests/api_tests/0box_free_storage_test.go (hunk 2 above)
Comment (shalinikum) [reply tests/api_tests/0box_free_storage_test.go:68]: Done.

File: tests/api_tests/config/api_tests_config.yaml (hunk 1 above)
Comment (shalinikum) [reply tests/api_tests/config/api_tests_config.yaml]: Done.

**Review by Kishan-Dhakan — APPROVED** [review]
LGTM

**Pull request merged**